extensions/amp-video/0.1/amp-video.js

```diff
@@ -262,6 +259,11 @@ export class AmpVideo extends AMP.BaseElement {
       'artwork': [{'src': artwork || poster || ''}],
     };
 
+    // Cached so mediapool operations (eg: swapping sources) don't interfere with this bool.
+    this.hasBitrateSources_ =
+      !!this.element.querySelector('source[data-bitrate]') ||
```

Comment:
Up to you, but we might want an early return if (mediaPool) so we have 0 impact on non stories doc

Reply:
Ok I see what you mean. We can have an early return if we end up doing more here, but this variable is only used on `resetOnDomChange` so it already has 0 impact on non-stories docs. Let's keep it to the simplest code possible so it's easy to understand and debug, but if we need to shortcircuit the expression at some point, we can do it.